CMakeLists.txt

```diff
@@ -1,6 +1,34 @@
-cmake_minimum_required(VERSION 3.6.2)
+cmake_minimum_required(VERSION 3.14)
 
-# Set the project name
 project(dotnet-monitor)
 
-add_subdirectory(src/external/runtime/src/coreclr)
+include(eng/native/configurepaths.cmake)
+include(${CLR_ENG_NATIVE_DIR}/configurecompiler.cmake)
+
+include_directories(
+    src/external/runtime/src/coreclr/pal/prebuilt/inc
+    src/inc
+    )
+
+if(CLR_CMAKE_HOST_UNIX)
+    include_directories(
+        src/external/runtime/src/coreclr/pal/inc/rt
```

Comment:
Is there a reason why we explicitly include a directory that is part of the directory on the next line?

Reply:
Subdirectories are not probed automatically. If this isn't done, then source code would have to include headers using something like `#include "rt/palrt.h"` rather than just doing `#include "palrt.h"`. I want to avoid updating the PAL file if absolutely avoidable.